README.md

```diff
@@ -215,7 +215,8 @@ Rspack and Rsbuild support most of the [unplugin](https://github.com/unplugin),
 - [rspress-plugin-clarity](https://github.com/jl917/rspress-plugin-clarity): Rspress plugin for [Clarity](https://clarity.microsoft.com/) integration.
 - [rspress-plugin-sitemap](https://github.com/jl917/rspress-plugin-sitemap): Automatically generate SEO-related sitemaps.
 - [rspress-plugin-mention-github](https://github.com/hunghg255/rspress-plugin-mention-github): Rspress plugin mention github user, or any link.
-
+- [rspress-plugin-vue](https://www.npmjs.com/package/rspress-plugin-vue): Rspress plugin is used to support Vue.
```

Comment:
Hi, can you provide the GitHub repo?

Reply:
If this plugin is not open source, I tend not to add it to the awesome-rspack repository. We can add it after the plugin is open source.

Reply:
get